cmake/kconfig.cmake

```diff
@@ -138,7 +138,7 @@ foreach(kconfig_target
     ZEPHYR_TOOLCHAIN_VARIANT=${ZEPHYR_TOOLCHAIN_VARIANT}
     ${COMMON_KCONFIG_ENV_SETTINGS}
     "SHIELD_AS_LIST=${SHIELD_AS_LIST_ESCAPED}"
-    EXTRA_DTC_FLAGS=${EXTRA_DTC_FLAGS}
```

Comment:
I believe this line can be completely removed.
It was originally introduced here https://github.com/zephyrproject-rtos/zephyr/commit/9aefdaff2ef893e303a67ae61f5ed3789d02ee61in order to share `EXTRA_DTC_FLAGS` with kconfigfunctions.py, but that need was removed later here https://github.com/zephyrproject-rtos/zephyr/pull/26589/commits/677b59a86dd0269dfc3a5d24cabc71856fbe5213 when edt is loaded from the file.

Reply:
Would like @mbolivar-nordic to chime in on that.

Reply:
That looks right to me.

Reply:
So should i remove the line containing EXTRA_DTC_FLAGS here completely or should i leave it as it is now?

Reply:
Should be removed completely.

Reply:
done